ports/esp32/machine_ulp.c

```diff
@@ -0,0 +1,100 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017 "Eric Poulsen" <eric@zyxod.com>
```

Comment:
Should replace Eric's name with your own

Reply:
jup i should. fixed.